src/test/scala/org/ergoplatform/nodeView/viewholder/ErgoNodeViewHolderSpec.scala

```diff
@@ -224,31 +225,6 @@ class ErgoNodeViewHolderSpec extends ErgoPropertyTest with NodeViewTestOps with
     getRootHash shouldBe Algos.encode(block1.header.stateRoot)
   }
 
-  private val t11 = TestCase("apply payload in incorrect order") { fixture =>
```

Comment:
why this test is removed?

Reply:
@catena2w , payloads could only be applied sequentially now (due to interlinks validation), so this test is outdated.